<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w2</class>
 <widget class="QWidget" name="w2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;Adobe 黑体 Std R&quot;;
color: rgb(200, 200, 200);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;直方图处理&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="styleSheet">
            <string notr="true">font: 15pt &quot;Adobe 黑体 Std R&quot;;
color: rgb(200, 200, 200);</string>
           </property>
           <property name="text">
            <string>类型</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBox_3">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(68, 68, 68);
font: 15pt &quot;Adobe 黑体 Std R&quot;;
color: rgb(200, 200, 200);</string>
           </property>
           <item>
            <property name="text">
             <string>直方图均衡化</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>直方图匹配</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(68, 68, 68);
font: 15pt &quot;Adobe 黑体 Std R&quot;;
color: rgb(200, 200, 200);</string>
     </property>
     <property name="text">
      <string>选择参考图像</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Adobe 黑体 Std R&quot;;
color: rgb(200, 200, 200);</string>
       </property>
       <property name="text">
        <string>路径：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Adobe 黑体 Std R&quot;;
color: rgb(200, 200, 200);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="ok">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(68, 68, 68);
font: 15pt &quot;Adobe 黑体 Std R&quot;;
color: rgb(200, 200, 200);</string>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(68, 68, 68);
font: 15pt &quot;Adobe 黑体 Std R&quot;;
color: rgb(200, 200, 200);</string>
       </property>
       <property name="text">
        <string>重置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(68, 68, 68);
font: 15pt &quot;Adobe 黑体 Std R&quot;;
color: rgb(200, 200, 200);</string>
       </property>
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>203</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(68, 68, 68);
font: 15pt &quot;Adobe 黑体 Std R&quot;;
color: rgb(200, 200, 200);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;直方图处理是一种点处理方法，它&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;通过对灰度直方图进行变换从而实&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;现图像增强。直方图处理主要有直&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;方图均衡化和直方图规定化两种方法。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>202</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
